src/com/gh4a/loader/IssueCommentListLoader.java

```diff
@@ -1,25 +1,27 @@
 package com.gh4a.loader;
 
-import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.Comparator;
-import java.util.List;
+import android.content.Context;
+
+import com.gh4a.Gh4Application;
 
 import org.eclipse.egit.github.core.Comment;
 import org.eclipse.egit.github.core.IssueEvent;
 import org.eclipse.egit.github.core.RepositoryId;
 import org.eclipse.egit.github.core.service.IssueService;
 
-import android.content.Context;
-
-import com.gh4a.Gh4Application;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
 
 public class IssueCommentListLoader extends BaseLoader<List<IssueEventHolder>> {
-    private final String mRepoOwner;
-    private final String mRepoName;
-    private final int mIssueNumber;
+    private final boolean mIsPullRequest;
+
+    protected final String repoOwner;
+    protected final String repoName;
+    protected final int issueNumber;
```

Comment:
No, m is for member variables. I generally follow the AOSP code style: http://source.android.com/source/code-style.html